content/cfp.html

```diff
@@ -0,0 +1,82 @@
+type: page
+title: Call for Proposals
+slug: cfp
+body_class_hack: talks
+---
+<h2>Call for proposals</h2>
+
+<h3>We want proposals from...</h3>
+<em>You.</em>
+
+<p>This is a <strong>community conference</strong>. You’re coming to the event, and that makes <strong>you</strong> part of the community and makes this <strong>your</strong> conference. If you’ve done something that you think other people might be interested in, or that you think they should know about, <em>we would love to know about it</em>.</p>
+
+<p>We <strong>don’t</strong> just want to hear from the experts or famous names - other perspectives are just as valuable, so don’t let the idea that you’re not expert or famous enough stop you because <em>this won’t stop us from selecting your talk or the audience from enjoying it</em>.</p>
+
+<p>Note that there is a <a href="#financial-support">programme of financial support</a> for speakers.</p>
+
+<h3>We <em>especially</em> want proposals from...</h3>
+
+<em>People who are minorities in this field</em>
+<p><a href="/diversity-accessibility">Diversity at PyCon</a> is important to us.</p>
+
+<p>If you can help us make this event more diverse just by being there, please, do come forward with a proposal. We need you.</p>
```

Comment:
Was never very happy with that, it does sound a bit checkboxy.

By the way, most of this content has simply been adopted from http://2015.djangocon.eu, hebce the occasional Django that I have missed.